integration_tests/src/main/python/udf_test.py

```diff
@@ -33,7 +34,7 @@
         raise AssertionError("incorrect pyarrow version during required testing " + str(e))
     pytestmark = pytest.mark.skip(reason=str(e))
 
-from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_collect
+from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_collect, assert_equal
```

Comment:
NIT: An useless import to assert_equal

Reply:
Yeah, good finding. Let's remove it in other  PR that affects this file.